src/app/api/statistics/route.js

```diff
@@ -18,11 +18,23 @@ export const GET = async () => {
   }
 
   try {
-    const [statistics, events] = await Promise.all([
+    const roles = [
```

Comment:
this shouldnt be a query on the entire table, it should be a form based update, check how the status gets updated